README.md

```diff
@@ -24,10 +24,10 @@ PhoneGap plugin for managing application assets with javascript asset maps. Incl
 
 ### downloadFile()
 
-**wizAssets.downloadFile(String URL, String filePathToBeStoredWithFilename, Function success, Function fail);**
+**wizAssets.downloadFile(String URL, String URI, Function success, Function fail);**
```

Comment:
Yes, a URI as Uniform Resource Identifier, which will be translated as a relative path.
A file URI is a URI scheme which must fit the specific form ```file://host/path```.
After a few headbangs at the actual source code, I made a few variable rename and clean up to match this convention, all included in the previous PR. It might sound confusing but it felt like the right way to go. The Android database field name was already using this naming too.